man/systemd.link.xml

```diff
@@ -409,6 +409,13 @@
           be ignored.</para>
         </listitem>
       </varlistentry>
+      <varlistentry>
+        <term><varname>TxQueueLength=</varname></term>
+        <listitem>
+          <para>Specifies the transmit queue length of the device in number of packets. An unsigned integer
+          in the range 0..4294967294. When unset, the kernel's default will be used.</para>
+        </listitem>
```

Comment:
https://elixir.bootlin.com/linux/v4.2/source/net/core/dev.c#L6911  range is 1..4294967294

Reply:
Thanks for reviewing, ssahani!
However I think that you are not correct: the kernel's function [`alloc_netdev_mqs()`](https://elixir.bootlin.com/linux/v5.11-rc3/source/net/core/dev.c#L10497) you have found is the one that is used to set the number of transmit/receive queues e.g. as a result of [having received an apropriate netlink message](https://elixir.bootlin.com/linux/v5.11-rc3/source/net/core/rtnetlink.c#L3141) (for example by utilizing the mechanics that you implemented in PR #17635). Sending a netlink message to change the transmit queue length ends in the kernel's function [`do_setlink()`](https://elixir.bootlin.com/linux/v5.11-rc3/source/net/core/rtnetlink.c#L2728) which itself calls [`dev_change_tx_queue_len()`](https://elixir.bootlin.com/linux/v5.11-rc3/source/net/core/dev.c#L8648). And there it is only checked for the range of an unsigned integer which is 0..4294967295.